webpack.config.js

```diff
@@ -45,7 +45,7 @@ const config = {
       },
       {
         test: /(\.tsx?$)/,
-        exclude: /node_modules/,
+        exclude: /node_modules\/(?!(@artsy\/reaction-force\/src\/__generated__)).*/,
```

Comment:
Why is this excluded?

Reply:
https://github.com/artsy/force/pull/2178#discussion_r168809291

Reply:
See my comment [here](https://github.com/artsy/force/pull/2178#discussion_r168814393).

Reply:
Thanks 👍